hubspot3/test/test_tickets.py

```diff
@@ -49,3 +57,207 @@ def test_get_all_tickets():
     assert tickets
     assert isinstance(tickets, list)
     assert len(tickets) > 1
+
+
+@pytest.mark.parametrize(
+    "extra_properties_given, extra_properties_as_list",
+    [
+        (None, []),
+        ("lead_source", ["lead_source"]),
+        (
+            ["hs_analytics_last_url", "hs_analytics_revenue"],
+            ["hs_analytics_last_url", "hs_analytics_revenue"],
+        ),
+    ],
+)
+def test_get_batch(
+    tickets_client,
+    mock_connection,
+    extra_properties_given,
+    extra_properties_as_list,
+):
+    response_body = {
+        "3234574": {"objectId": 3234574, "properties": {}},
+        "3234575": {"objectId": 3234575, "properties": {}},
+    }
+    ids = ["3234574", "3234575"]
+    properties = tickets_client.__default_batch_properties.copy()
+    properties.extend(extra_properties_as_list)
+    params = {"properties": p for p in properties}
```

Comment:
Is this correct? If I'm reading this right, `properties` is a list (as indicated by the usage of `.extend()`. If for example `properties = ['a', 'b', 'c']` and you use `params = {'properties': p for p in properties}`, then your params will be set to `params = {'properties': 'c'}`.

Reply:
I was following the other examples, thinking that would bring the list. Now testing I discovered that the idea was to get one element, the lower function that tests the parameters only accepts one check per properties.
Going to fix it or check for a work around